### PR TITLE
Year Story List, Upload Form, Media fixes

### PR DESCRIPTION
**Pull request by shekharkhedekar**

### CODE REVIEW

File: wp-content/themes/vanguard-history/front-page.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ?>
 
 <main id="primary" class="site-main">
-    <div class="content-section content-intro">
+    <div class="content-section content-primary">
```
Comment (mrengy) [inline wp-content/themes/vanguard-history/front-page.php:19]: Nice convention here.

**Review by mrengy — APPROVED** [review]

**Comment by mrengy** [comment]
Looking good! I deployed this into dev and stage.

One thing to note - I initially got old styling when I refreshed [the year stories page](https://historyscv-dev.dreamhosters.com/year_story/) in dev. Doing a shift-refresh loaded the new styles. Was the thing you were working on for this supposed to prevent this issue? Not a big deal at all though.

old styles on dev site

**Comment by shekharkhedekar** [comment]
Yes! Forgot to bump the version. Let me think about an automated way to do this.

**Comment by mrengy** [comment]
It’s not a problem we need to solve right now. I’d say don’t worry about it if it is not a straightforward thing. 

--
Mike Eng (he/him)
mike-eng.com linkedin.com/in/mikeeng401

**Pull request merged**